### PR TITLE
Active record association

### PR DESCRIPTION
**Pull request by chitrak-bhatt**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+    helper_method :logged_in?
+    def current_myuser
+        Myuser.find_by(id: session[:myuser_id])
+    end
+
+    def logged_in?
+        !current_myuser.nil?
+    end
+
+    # def authorized
```
Comment (dhruvika29kevat) [inline app/controllers/application_controller.rb]: Remove commented code if it is of no use

File: app/views/categories/update.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Categories#update</h1>
```
Comment (dhruvika29kevat) [inline app/views/categories/update.html.erb]: Remove unnecessary html files.

File: app/views/sessions/create.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#create</h1>
```
Comment (dhruvika29kevat) [inline app/views/sessions/create.html.erb]: Remove Remove unnecessary html files.

File: app/views/sessions/login.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#login</h1>
```
Comment (dhruvika29kevat) [inline app/views/sessions/login.html.erb]: Remove unnecessary html files.

**Review by dhruvika29kevat — COMMENTED** [review]
Which comment functionality is working ?
From comments controller or the methods defined in events controller?

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (chitrak-bhatt) [reply app/controllers/application_controller.rb]: > Which comment functionality is working ? From comments controller or the methods defined in events controller?
From controller it is working will remove the code from events controller.

**Pull request merged**